src/slider/tp-slider.ts

```diff
@@ -15,6 +15,9 @@ export class TPSliderElement extends HTMLElement {
 	 * Properties.
 	 */
 	protected touchStartX: number = 0;
+	private responsiveSettings: TPSliderSettings;
+	private defaultSettings: TPSliderSettings;
+	private settingKeys: string[];
```

Comment:
@imranhsayed We need to be careful in setting these variables:

1. Lets change them to `protected` - that way any inheriting classes can access / override them
2. Lets prefix with `responsive` so we know "what" settings these are. Example: `protected allowedResponsiveKeys`